Tests/HummingbirdRedisTests/PersistTests.swift

```diff
@@ -14,229 +14,426 @@
-        try app.XCTExecute(uri: "/persist/\(tag)", method: .GET) { response in
-            XCTAssertEqual(response.status, .noContent)
+    func testRemove() async throws {
+        let (router, _) = try createRouter()
+        let app = HBApplication(responder: router.buildResponder())
+        try await app.test(.router) { client in
+            let tag = UUID().uuidString
+            try await client.XCTExecute(uri: "/persist/\(tag)", method: .put, body: ByteBufferAllocator().buffer(string: "ThisIsTest1")) { _ in }
+            try await client.XCTExecute(uri: "/persist/\(tag)", method: .delete) { _ in }
+            try await client.XCTExecute(uri: "/persist/\(tag)", method: .get) { response in
+                XCTAssertEqual(response.status, .noContent)
+            }
         }
     }
 
-    func testExpireAndAdd() throws {
-        let app = try createApplication()
-        try app.XCTStart()
-        defer { app.XCTStop() }
+    func testExpireAndAdd() async throws {
+        let (router, _) = try createRouter()
+        let app = HBApplication(responder: router.buildResponder())
+        try await app.test(.router) { client in
 
-        let tag = UUID().uuidString
-        try app.XCTExecute(uri: "/persist/\(tag)/0", method: .PUT, body: ByteBufferAllocator().buffer(string: "ThisIsTest1")) { _ in }
-        Thread.sleep(forTimeInterval: 1)
-        try app.XCTExecute(uri: "/persist/\(tag)", method: .GET) { response in
-            XCTAssertEqual(response.status, .noContent)
-        }
-        try app.XCTExecute(uri: "/persist/\(tag)/10", method: .PUT, body: ByteBufferAllocator().buffer(string: "ThisIsTest1")) { response in
-            XCTAssertEqual(response.status, .ok)
-        }
-        try app.XCTExecute(uri: "/persist/\(tag)", method: .GET) { response in
-            XCTAssertEqual(response.status, .ok)
-            let body = try XCTUnwrap(response.body)
-            XCTAssertEqual(String(buffer: body), "ThisIsTest1")
+            let tag = UUID().uuidString
+            try await client.XCTExecute(uri: "/persist/\(tag)/0", method: .put, body: ByteBufferAllocator().buffer(string: "ThisIsTest1")) { _ in }
+            try await Task.sleep(nanoseconds: 1_000_000_000)
+            try await client.XCTExecute(uri: "/persist/\(tag)", method: .get) { response in
+                XCTAssertEqual(response.status, .noContent)
+            }
+            try await client.XCTExecute(uri: "/persist/\(tag)/10", method: .put, body: ByteBufferAllocator().buffer(string: "ThisIsTest1")) { response in
+                XCTAssertEqual(response.status, .ok)
+            }
+            try await client.XCTExecute(uri: "/persist/\(tag)", method: .get) { response in
+                XCTAssertEqual(response.status, .ok)
+                let body = try XCTUnwrap(response.body)
+                XCTAssertEqual(String(buffer: body), "ThisIsTest1")
+            }
         }
     }
+    /*        let app = HBApplication(testing: .live)
```

Comment:
Lots of commented tests still

Reply:
Removed. All relevant commented tests have equivalent versions implemented